### ISSUE TITLE
Pydeps returning blank picture

### ISSUE THREAD

**Issue by anipan** [thebjorn/pydeps#91]
Hello,

I am trying to create a dependency graph for a package with multiple python files in a folder. When I try to run pydeps on any of the files, it returns a blank picture. Do I have to add more arguments for the python module that I am trying to run through pydeps? If so, how can I do this?

I apologize if this question is a bit unclear or has already been answered; I am quite new to this.

**Comment by thebjorn** [comment]
Given the following structure:
```
(dev35) go|c:\srv\tmp\issue91> yamldirs myapp
myapp:
  a.py: from . import b
  __init__.py: ''
  b.py: ''
```
i.e. a subdirectory `myapp` with 3 files: `__init__.py` (empty), `b.py` (empty), and `a.py` containing the line

    from . import b

the following `pydeps` invocation (i.e. current directory is the parent directory, calling `pydeps` with the name of the package folder):

    (dev35) go|c:\srv\tmp\issue91> pydeps myapp

produces the following graph:

![myapp](https://user-images.githubusercontent.com/1193143/118790686-4c524f80-b896-11eb-8548-2b4a044b6354.png)

**Comment by machow** [comment]
I've noticed that if there are issues with imports (e.g. the code tries to import something that doesn't exist), then it sometimes returns an empty graph. For example, if in your above example you replaced `a.py`...

* old: `from . import b`
* new: `import b`

The graph is blank.

**Comment by adnion** [comment]
`The graph is blank.`
Same for me and hard to debug because `--debug` flag does not output anything

**Comment by thebjorn** [comment]
To include packages that are not installed, you'll need the `--include-missing` flag. To debug problems with the module finding, use the `--debug-mf <int>` flag (I would suggest starting with 2 as the int-value.. the output is voluminous).

**Comment by open768** [comment]
I'm having the same problem, with a blank being shown. Perhaps when this happens, pydeps should put a helpful warning message to the console saying that it couldnt produce a diagram and where the user should go for help

**Comment by thebjorn** [comment]
@open768

**Comment by purva-thakre** [comment]
@thebjorn Hi, thanks for this wonderful project! I came across the closed issue because I also got the same output. 

I am trying to analyze [toqito](https://github.com/vprusso/toqito) through pydeps. Unfortunately, when I run `pydeps toqito` in the main `toqito` directory, I get a blank image. But if I run `pydeps toqito/channels` where `channels` is [a module](https://github.com/vprusso/toqito/tree/master/toqito/channels) in toqito, I get the expected image only for the `channels` module. 

Is there a reason why it works on the project's modules but not in the main directory?

**Comment by thebjorn** [comment]
@purva-thakre you should be in the directory with the README.md file when running `pydeps toqito` ...

**Comment by purva-thakre** [comment]
> [@purva-thakre](https://github.com/purva-thakre) you should be in the directory with the README.md file when running `pydeps toqito` ...

Yup, that's what I did, and it gave me a blank image, which is why I tried running it in the modules within the project.

**Comment by thebjorn** [comment]
Ah. toqito isn't a "module". I'm sure it does some python magic during install to make it a module, but if you just want to get pydeps working you can add an empty `__init__.py` file in the `toqito/toqito` directory.

**Comment by purva-thakre** [comment]
> but if you just want to get pydeps working you can add an empty __init__.py file in the toqito/toqito directory.

Thanks! This worked.
